lighthouse-core/report/html/templates.html

```diff
@@ -374,14 +374,14 @@
       </button>
       <div class="lh-tools__dropdown">
          <!-- TODO(i18n): localize tools dropdown -->
-        <a href="#" class="report-icon report-icon--print" data-action="print-summary">Print Summary</a>
-        <a href="#" class="report-icon report-icon--print" data-action="print-expanded">Print Expanded</a>
-        <a href="#" class="report-icon report-icon--copy" data-action="copy">Copy JSON</a>
-        <a href="#" class="report-icon report-icon--download" data-action="save-html">Save as HTML</a>
-        <a href="#" class="report-icon report-icon--download" data-action="save-json">Save as JSON</a>
-        <a href="#" class="report-icon report-icon--open lh-tools--viewer" data-action="open-viewer">Open in Viewer</a>
-        <a href="#" class="report-icon report-icon--open lh-tools--gist" data-action="save-gist">Save as Gist</a>
-        <a href="#" class="report-icon report-icon--dark" data-action="toggle-dark">Toggle Dark Theme</a>
+        <a href="#" class="report-icon report-icon--print" id="lh-dropdown_print_summary" data-action="print-summary">Print Summary</a>
```

Comment:
feels like these ids should be one of

- `lh-dropdown_print-summary`
- `lh-dropdown__print-summary`
- `lh-dropdown__print_summary`

thoughts?

Reply:
I like `lh-dropdown__<text>-<text>` it's like kebab-snake__kebab-case